commit 6ff98b962db15d18eb1d082fe344cef692ecef8e
Author: Sergei Trofimovich <slyfox@gentoo.org>
Date:   Thu Apr 6 08:55:56 2017 +0100

    config.mk.in: remove phase=0 hack for CrossCompilePrefix
    
    $(CrossCompilePrefix) is used only in 'make install'
    target filenames in $(DESTDIR). None of inplace (or boot)
    files contain $(CrossCompilePrefix).
    
    Thus we don't need to worry about phases.
    
    Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>

diff --git a/mk/config.mk.in b/mk/config.mk.in
index 4e61eea821..4d5d82aa80 100644
--- a/mk/config.mk.in
+++ b/mk/config.mk.in
@@ -530,11 +530,7 @@ SUPPORTS_THIS_UNIT_ID = @SUPPORTS_THIS_UNIT_ID@
 # needs to know which gcc you're using in order to perform its tests.
 
 GccVersion            = @GccVersion@
-ifeq "$(phase)" "0"
-CrossCompilePrefix    =
-else
 CrossCompilePrefix    = @CrossCompilePrefix@
-endif
 # TargetPlatformFull retains the string passed to configure so we have it in
 # the necessary format to pass to libffi's configure.
 TargetPlatformFull    = @TargetPlatformFull@
